src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigForm.tsx

```diff
@@ -107,20 +123,24 @@ export const AutomatedAnalysisConfigForm: React.FC<AutomatedAnalysisConfigFormPr
   const [maxSize, setMaxSize] = React.useState(recordingConfig.maxSize);
   const [maxSizeUnits, setMaxSizeUnits] = React.useState(1);
   const [errorMessage, setErrorMessage] = React.useState('');
-  const [isLoading, setIsLoading] = React.useState(true);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [editing, setEditing] = React.useState(false);
 
   const refreshTemplates = React.useCallback(() => {
     setIsLoading(true);
     addSubscription(
-      (targetObs ? targetObs : context.target.target())
+      context.target
```

Comment:
Can we set up as follow: 
- A Target Subject as a ref.
- Component take a Target prop.
- Register an effect that any time this prop change -> cause the Subject to emit this new Target prop.
- Here, listen to that Subject instead

See reference in Topology:

https://github.com/cryostatio/cryostat-web/blob/96ec5106581c4fd9f8adec26177fbe13f60962d8/src/app/Topology/Shared/Entity/utils.tsx#L387

Reply:
Parent here means: `AutomatedAnalysisConfig.tsx`, `AutomatedAnalysis/AutomatedAnalysisConfigDrawer.tsx`, and `AutomatedAnalysis/AutomatedAnalysisCard.tsx (advancedConfig)` (this will need a wrapper).